docs/classic-ui/portlets.md

```diff
@@ -1,13 +1,185 @@
 ---
 myst:
   html_meta:
-    "description": ""
-    "property=og:description": ""
-    "property=og:title": ""
-    "keywords": ""
+    "description": "Basic information about portlets in classic UI"
+    "property=og:description": "Basic information about portlets in classic UI"
+    "property=og:title": "Portlets"
+    "keywords": "portlets"
 ---
 
 (classic-ui-portlets-label)=
 
 # Portlets
 
+## What is a Portlet?
+
+In Plone, a portlet is a small, modular piece of content that can be displayed in a specific area of a web page. Portlets are typically used to display information that is relevant to the current context, such as the latest news, upcoming events, or a list of related documents.
+
+The context is either the current part of the content hierarchy, the current user's group memberships, or the current content type.
+Thus, if a portlet was set on a folder, all contained items do display the portlet unless it is explicitly blocked.
+
+Portlets are highly customizable and can be used to display a wide variety of information.
+They can be added, removed, or rearranged on a web page by users with the appropriate permissions, allowing for a high degree of flexibility in the layout and content of a Plone site.
+
+Plone comes with several built-in portlets, such as the news portlet, the events portlet, and the login portlet.
+In addition, developers can create custom portlets to display specific types of information or to provide specific functionality.
+
+## Adding a portlet to a page
+
+As a user, you can add a portlet to a web page in a Plone site by following these steps:
+
+1. Navigate to the web page where you want to add the portlet.
+
+2. Click on the "Manage portlets" link in the toolbar of the page and select the region on the page to modify.
+   This will open the "Manage portlets" screen.
+
+3. In the "Add portlets" menu, select the portlet that you want to add and click the "Add" button.
```

Comment:
For menus, it might be better to use [`menuselection`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection) over [`guilabel`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-guilabel).

```suggestion
3. In the {menuselection}`Add portlets` menu, select the portlet that you want to add, and click the {guilabel}`Add` button.
```